common-css/documentation.css

```diff
@@ -0,0 +1,83 @@
+.document-container {
+  display: grid;
+  grid-template-columns: 20% 80%;
+  font-size: var(--f5);
+  margin-top: var(--s4);
+  padding: 0 5%;
+}
+
+.component-list {
+  display: flex;
+  flex-direction: column;
+  font-family: "Playfair", sans-serif;
+  padding-left: var(--s2);
+}
+.sidebar li {
+  list-style: none;
+}
+
+.component-list a {
+  text-decoration: none;
+  color: var(--gun-metal);
+  margin: var(--s1) 0;
+  border-radius: var(--s2);
+}
+.component-list a:hover {
+  color: var(--primary);
+  background-color: var(--secondary);
+}
+
+.component-list a.active-item {
+  color: var(--primary);
+  background-color: var(--text);
+}
+
+.document-main {
+  font-family: "Playfair", sans-serif;
```

Comment:
I think font-family is inherited so we can set font-family in the body. We need not to redeclare it again